packages/web3-eth/test/integration/subscribe.test.ts

```diff
@@ -26,20 +31,23 @@ describe('unsubscribe', () => {
 			const subs = web3Eth?.subscriptionManager?.subscriptions;
 			const inst = subs?.get(Array.from(subs.keys())[0]);
 			expect(inst).toBeInstanceOf(NewHeadsSubscription);
+			await web3Eth.clearSubscriptions();
```

Comment:
This should be included in an [afterEach](https://jestjs.io/docs/setup-teardown)